packages/ui-tray/src/Tray/index.tsx

```diff
@@ -194,8 +179,7 @@ class Tray extends Component<TrayProps> {
         {portalIsOpen && (
           <Transition
             in={open}
-            // @ts-expect-error ts-migrate(2769) FIXME: No overload matches this call.
-            type={this.transition}
+            type={this.transition as TransitionType}
```

Comment:
It is better to type the getter, it just needs a little help so it knows what it returns:

```ts
get transition(): TransitionType {
    return `slide-${this.direction}`
  }
```